contracts/stake/StakedPSYS.sol

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
Any specific reason whey you decide to go with version 1 rather than 3? It seems that v3 has some delegation methods that are not available in v1

Reply:
I noticed that the deploy repository depends on it, and the deployment script initially deploys version 1, followed by upgrades to versions 2 and then 3.

Reply:
> Any specific reason whey you decide to go with version 1 rather than 3? It seems that v3 has some delegation methods that are not available in v1

I noticed that the deploy repository depends on it, and the deployment script initially deploys version 1, followed by upgrades to versions 2 and then 3.